File: product/src/app/model/therapist.ts
export class Therapist {
    id: number = 0;
    firstName: string = '';
    lastName: string = '';
    specialization: string = '';
    phone: string = '';
    email: string = '';
    available: boolean = true;
    
    get fullName(): string {
        return `${this.firstName} ${this.lastName}`;
    }
}

File: ecommerce/src/main/java/com/nicco/model/Therapist.java
package com.nicco.model;

import lombok.Data;
import java.util.Date;

@Data
public class Therapist {
    private int id;
    private String firstName;
    private String lastName;
    private String specialization;
    private String phone;
    private String email;
    private boolean available;
    private Date created;
    private Date lastUpdated;
    
    public String getFullName() {
        return firstName + " " + lastName;
    }
}